packages/ui-core/shared/src/lib/components/changelog-entry/changelog-entry.component.html

```
@@ -0,0 +1,24 @@
+	<div class="entry-headings-wrapper">
+		<nb-icon [icon]="entry.icon" class="icon"></nb-icon>
+		<div class="entry-headings">
+			<div class="entry-header">{{ entry.title }}</div>
+			<div class="entry-header-date">{{ entry.date | date }}</div>
+		</div>
+	</div>
+	<p class="paragraph">
+		{{ entry.content }}
+	</p>
+	@if (entry.imageUrl) {
+		<img class="entry-img" [src]="entry.imageUrl" [alt]="entry.title" />
```

HTMLHint (1.9.2)

[warning] An alt attribute must be present on <img> elements.

(alt-require)